go/stellar/remote/remote.go

```diff
@@ -1260,3 +1260,38 @@ func ServerTimeboundsRecommendation(ctx context.Context, g *libkb.GlobalContext)
 	}
 	return res.TimeboundsRecommendation, nil
 }
+
+func SetInflationDestination(ctx context.Context, g *libkb.GlobalContext, signedTx string) (err error) {
+	apiArg := libkb.APIArg{
+		Endpoint:    "stellar/setinflation",
+		SessionType: libkb.APISessionTypeREQUIRED,
+		Args: libkb.HTTPArgs{
+			"sig": libkb.S{Val: signedTx},
+		},
+		MetaContext: libkb.NewMetaContext(ctx, g),
+	}
+	_, err = g.API.Post(apiArg)
+	return err
+}
+
+type getInflationRes struct {
+	libkb.AppStatusEmbed
+	DestinationAccountID *stellar1.AccountID `json:"destination_account_id"`
+}
+
+func GetInflationDestination(ctx context.Context, g *libkb.GlobalContext, accountID stellar1.AccountID) (res *stellar1.AccountID, err error) {
```

Comment:
will this also be in stellar1.AccountDetails?

Reply:
there is no reason not to put it there, right?

Reply:
If it's there, then you can do something like `walletState.Details(ctx, accountID).InflationDestination`